src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1707,6 +1701,11 @@ private TokenCacheItem findFamilyItemForUser(final UserIdentifierType userIdenti
         Logger.v(TAG + methodName, "No family items found in the cache.");
         return null;
     }
+
+    private boolean isRefreshTokenForFamily(TokenCacheItem tokenCacheItem) {
+        return (!StringExtensions.IsNullOrBlank(tokenCacheItem.getFamilyClientId())
```

Comment:
nit: consider adding a log message here on failure, else from logs you/user will not be able to tell if check in 1686 or 1687 failed